python/hail/api2/matrixtable.py

```diff
@@ -1777,12 +1774,13 @@ def joiner(left):
                 raise NotImplementedError('vds join with indices {}'.format(indices))
 
             return construct_expr(Select(Reference(prefix), uid),
-                                  self.row_schema, indices, aggregations, joins.push(Join(joiner, uids_to_delete)))
+                                  self.row_schema, indices, aggregations,
```

Comment:
just making sure you don't mean to use construct_reference here and below

Reply:
nope, references are meant to be top-level fields used and this is a UID